src/components/Anchor/index.js

```diff
@@ -21,13 +21,14 @@ const prefixByType = {
 const Anchor = ({
   url,
   children,
+  className = '',
   isSeparateTab = false,
   type = 'inline',
 }) => (
   <a
     href={url}
     target={isSeparateTab ? '_blank' : '_self'}
-    className={classNameByType[type]}
+    className={`${classNameByType[type]} ${className}`}
```

Comment:
By separation of concerns I mean, if a component truly needs to have `className` on it, that makes sense to add it - like if you are trying to change the text color of a `<Anchor>` to be green or whatever; but with padding/layout, I believe this belongs in a generic wrapper element.